blockchain/difficulty.go

```diff
@@ -286,7 +287,8 @@ func (b *BlockChain) calcNextRequiredDifficulty(lastNode *blockNode, newBlockTim
 
 	// Limit the amount of adjustment that can occur to the previous
 	// difficulty.
-	actualTimespan := lastNode.timestamp.UnixNano() - firstNode.timestamp.UnixNano()
```

Comment:
@stevenroose:  They all hold seconds.  The end result is not changed because it's a ratio between the adjusted timespam and the target timespan and both are in seconds.

I'm not sure what you mean that `b.minRetargetTimespan` isn't changed.  It is defined as:

```Go
minRetargetTimespan: targetTimespan / adjustmentFactor,
```
 And `targetTimespan` was updated to:

```Go
targetTimespan := int64(params.TargetTimespan / time.Second)
```